plugins/inputs/mysql/mysql.go

```diff
@@ -24,6 +30,10 @@ var sampleConfig = `
   #
   # If no servers are specified, then localhost is used as the host.
   servers = ["tcp(127.0.0.1:3306)/"]
+  PerfEventsStatementsDigestTextLimit = 120
```

Comment:
@Maksadbek Is there a way to modify the MySQL docker image to be a slave? Or at least act like one?